startup/81-dexela.py

```diff
@@ -0,0 +1,62 @@
+import os
+import ophyd
+from hxntools.detectors.dexela import (DexelaDetector, DexelaTiffPlugin)
+from ophyd import (AreaDetector, CamBase, TIFFPlugin, Component as Cpt,
+                    HDF5Plugin, Device, StatsPlugin, ProcessPlugin,
+                    ROIPlugin, EpicsSignal)
+from databroker.assets.handlers import HandlerBase
+from ophyd.areadetector.filestore_mixins import (FileStoreIterativeWrite,
+                                                 FileStoreHDF5IterativeWrite,
+                                                 FileStoreTIFFSquashing,
+                                                 FileStoreTIFF,
+                                                 FileStoreHDF5, new_short_uid,
+                                                 FileStoreBase
+                                                 )
+from ophyd.areadetector import (AreaDetector, PixiradDetectorCam, ImagePlugin,
+                                TIFFPlugin, StatsPlugin, HDF5Plugin,
+                                ProcessPlugin, ROIPlugin, TransformPlugin,
+                                OverlayPlugin)
+from ophyd.areadetector.trigger_mixins import SingleTrigger
+from enum import Enum
+
+
+class XPDDMode(Enum):
+    step = 1
+    fly = 2
+
+
+class XPDDDexelaTiffPlugin(TIFFPlugin, FileStoreTIFF, Device):
```

Comment:
Should this match the Plugin at PDF? https://github.com/NSLS-II-PDF/profile_collection/blob/master/startup/80-areadetector.py#L130

Reply:
No because I do not think we want to do squashing here.

Reply:
I think we want to be using `FileStoreTIFFIterativeWrite` https://github.com/NSLS-II/ophyd/blob/6cab3d2f2adbca2e4aab557bda80c1e2db175bfb/ophyd/areadetector/filestore_mixins.py#L652
rather that `FileStoreTIFF` here.

This will fix the missing `point_number` in the datum kwargs (as a `generate_datum` method is provided by `FileStoreIterativeWrite`).